src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -224,13 +225,13 @@ class ObjectLocker : public StackObj {
   Handle      _obj;
   BasicLock   _lock;
   NoPreemptMark _npm;
+  bool    _skip_exit;
  public:
-  ObjectLocker(Handle obj, JavaThread* current);
+  ObjectLocker(Handle obj, TRAPS);
```

Comment:
I wonder if we should declare `PREEMPTABLE_TRAPS` as an indicator that the only exception expected to come out of a call is the preempted-exception?

Reply:
Never mind - I thought there were a few functions that had TRAPS added just for the preempt case but now I don't see them. Thanks